docs/references/subcommands/analyze.md

```diff
@@ -114,14 +114,15 @@ We support the following archive formats:
   - `xz` compression
   - `zstd` compression
 
-### Enabling additional strategies
+### Enabling or disabling additional strategies
 
 In addition to the [standard flags](#specifying-fossa-project-details), the analyze command supports the following additional strategy flags:
 
 | Name                                                             | Description                                                                                                                                                              |
 |------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [`--detect-vendored`](./analyze/detect-vendored.md)              | Enable the vendored source identification engine. For more information, see the [C and C++ overview](../strategies/languages/c-cpp/c-cpp.md).                            |
-| [`--detect-dynamic './some-binary`](./analyze/detect-dynamic.md) | Analyze the binary at the provided path for dynamically linked dependencies. For more information, see the [C and C++ overview](../strategies/languages/c-cpp/c-cpp.md). |
+| [`--detect-vendored`](./analyze/detect-vendored.md)                               | Enable the vendored source identification engine. For more information, see the [C and C++ overview](../strategies/languages/c-cpp/c-cpp.md).                            |
+| [`--detect-dynamic './some-binary`](./analyze/detect-dynamic.md)                  | Analyze the binary at the provided path for dynamically linked dependencies. For more information, see the [C and C++ overview](../strategies/languages/c-cpp/c-cpp.md). |
+| [`--static-only-analysis`](../strategies/README.md#static-and-dynamic-strategies) | Do not use third-party tools when analyzing projects.
```

Comment:
[curious] why did we decide on this flag over `--strict`?

Reply:
I thought it was descriptive.

We also have a ticket to implement a `--strict` mode. I thought the goal for that would be to turn all warnings into errors and constrain the CLI to using only the "best" tactic for a given strategy. See this [ticket](https://fossa.atlassian.net/browse/ANE-236). I think we should reserve `--strict` for later. 

If you meant to write `--static` I think that would be fine. An invocation would be like `fossa analyze ... --static ...` which describes that it's the analysis that would be made static.

Reply:
oh wait you're right, I was just misremembering what `--strict` was meant to be!